DHL/JSON/66da0fe7-7a0b-4e36-b685-8f4d6b06607f.json

```diff
@@ -0,0 +1,531 @@
+                                    }
+                                  ],
+                                  "Rul": [
+                                    {
+                                      "RulId": "55b3c76f-cf7e-4a24-abeb-57694b7cd4ef",
+                                      "Nm": "Rule condition to Greek vat number",
+                                      "Cond": "**.Att[Id='6a2f893d-8c39-42ed-9b9e-450d21092907'].Val='Opt1'",
+                                      "AttrMtdt": [
+                                        {
+                                          "isMan": true,
+                                          "isHidden": false
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "471e0fbd-42e4-452c-b8da-1a47a0c82d3e",
+                                  "Nm": "Purpose Name - Greek VAT number",
+                                  "Dsc": "Purpose Name - Greek VAT number desc",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Agree to have Greek VAT number",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "Rul": [
+                                    {
+                                      "RulId": "e9082d77-d449-427e-b7cc-bdbd1101a8c6",
+                                      "Nm": "Not agree purpose",
+                                      "Cond": "**.Att[Id='0da3ad86-cc40-44ff-9ad4-60780be50fc8'].Val='Opt2'",
+                                      "AttrMtdt": [
+                                        {
+                                          "DflVal": "Not Agree to have Greek VAT number"
+                                        }
+                                      ]
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "b5438bd1-82cd-4617-a280-f1cb172a0128",
+                                  "Nm": "Purpose Description - Greek VAT number",
+                                  "Dsc": "Purpose Description - Greek VAT number desc",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Not Agree to have Greek VAT number",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "Rul": [
+                                    {
+                                      "RulId": "ed7d5c64-cab3-4b19-a4c7-13825ac3ee3d",
+                                      "Nm": "Not agree Do you have a Greek VAT number Description",
+                                      "Cond": "**.'0da3ad86-cc40-44ff-9ad4-60780be50fc8'].Val='Opt1'",
```

Comment:
add second / first radion button is condition also.
Example - >  "Cond": "**.Att[Id='6a2f893d-8c39-42ed-9b9e-450d21092907'].Val='Opt1' and **.Att[Id='0da3ad86-cc40-44ff-9ad4-60780be50fc8'].Val='Opt1'"

Reply:
Done